src/main/java/pl/jakubkozlowski/learning/firststeps/converter/ModelMapperConverter.java

```diff
@@ -40,4 +41,10 @@ default Entity convertDTO(DTO dto) {
             return dTOList.stream().map(this::convertDTO).collect(Collectors.toList());
     }
 
+    default Page<DTO> convertPageEntity(Page<Entity> entityPage) {
+        if (entityPage == Page.empty()) {
```

Comment:
Also, look at the spring map for Page. you could do something like that. More generic in my opinion. More elegant